Sources/Shared/API/Responses/Logbook.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import ObjectMapper
+
+public class LogbookResponse: Mappable {
+    public var Domain: String?
```

Comment:
similarly, let's stick with more swifty naming of new things -- variable names should start with a lowercase letter

Reply:
Done in 7501e2c5e56b2281923b97370309d4040f00f47b
I was wondering we do we use .NET style for part of the codebase. 

I left the style in `HomeAssistantAPI` as is rest of the class since I think this class would be best to refactor all methods at once.

Reply:
The .NET-style function naming is largely older code; renaming as you go is a fine approach, and keeping that API file consistent is good for now.